pepper-apis/studybuilder-cli/studies/singular/patches/ddp-8554-parental-validations.conf

```diff
@@ -0,0 +1,84 @@
+{
+  "validations": [
+
+    # When using the Enroll Child flow, if a user enters an age between >=7 years and AOM for their region
+    # and answers NO to cognitive-impairment in the pre-qual but enters a date of birth that equates to <7 years
+    # in the child consent, an error message should appear under the consent DOB field
+    # Formally: PREQUAL.AGE > 7 && !PREQUAL.CAPABLE && CONSENT.AGE <= 7
```

Comment:
Validations are not referring to PREQUAL . Its referring to AGE entered in ADD_PARTICIPANT_PARENTAL

Reply:
Same here

Reply:
Format criteria changed